core/modules/widgets/transclude.js

```diff
@@ -97,13 +99,36 @@ TranscludeWidget.prototype.makeRecursionMarker = function() {
 	output.push("}");
 	return output.join("");
 };
+	
+TranscludeWidget.prototype.hasSourceTextChanged = function() {
+	var tiddler,
+			parserType,
+		sourceText = null;
+	if(this.transcludeSubTiddler) {
+		tiddler = this.getSubTiddler(this.transcludeTitle,this.transcludeSubTiddler);
+	} else {
+		tiddler = this.wiki.getTiddler(this.transcludeTitle);
+	}
+	if(tiddler) {
+		if(!this.transcludeIndex) {
```

Comment:
As soon, as there is an "index" parameter, this code path will be executed. ... BUT the [widget docs](https://tiddlywiki.com/#TranscludeWidget) says, that the "field" parameter takes precedence, if it exists.